Entities/Common/Scripts/AutoGrabFromGround.as

```diff
@@ -17,7 +19,11 @@ void onTick(CBlob@ this)
 		for (uint i = 0; i < blobsInRadius.length; i++)
 		{
 			CBlob @b = blobsInRadius[i];
-			if (b.isOnGround() && !b.isAttached() && b.getName() == this.get_string("autograb blob"))
+
+			string[] autograb_blobs;
+			this.get("autograb blobs", autograb_blobs);
```

Comment:
This should be moved into onInit underneath the code that sets it if it doesn't exist.

Reply:
resolving this as we decided with eps on discord